sdk/communication/Azure.Communication.NetworkTraversal/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-02-22-preview1)
```

Comment:
It should be the release date, (I believe you added the API version here?).. so in your case it should be 2021-05-24